sdks/go/go_sdk.go

```diff
@@ -874,6 +878,7 @@ type condition struct {
 
 // handleCondition is a wrapper for inspecting the step condition and potentially
 // performing a notification and injecting metadata back into the response.
+// TODO: simplify these params
```

Comment:
Agreed - this is getting really long. I have to work on the SDK next, I'll do this.